### PR TITLE
Created new user view for account creation

### PR DESCRIPTION
**Pull request by itsaliu90**
new-account

Made using Licecap: http://licecap.en.softonic.com/

### CODE REVIEW

**Comment by carolynyoo** [comment]
LEGIT GIF

File: browser/js/common/directives/navbar/navbar.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 'use strict';
 app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state) {
 
-console.log("Boo!")
```
Comment (itsaliu90) [inline browser/js/common/directives/navbar/navbar.js:4]: Swry, remove that shiz

**Comment by darksim735** [comment]
Cool

**Comment by carolynyoo** [comment]
:ok_woman:

**Pull request merged**